modules/nf-core/toulligqc/tests/main.nf.test

```diff
@@ -0,0 +1,87 @@
+nextflow_process {
+
+    name "Test Process TOULLIGQC"
+    script "../main.nf"
+    process "TOULLIGQC"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "toulligqc"
+
+    test("sarscov2 - nanopore sequencing_summary") {
+        when {
+            process {
+                """
+                input[0] = [
+                    file(params.test_data['sarscov2']['nanopore']['test2_sequencing_summary'], checkIfExists: true)
+                    ]
+                input[1] = [
+                    ]
+                input[2] = [
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(file(process.out.report_data[0]).readLines()[11..74]).match() },
+                { assert process.out.report_html[0] ==~ ".*/report.html"}
+            )
+        }
+
+    }
+
+    test("sarscov2 - nanopore fastq") {
+        when {
+            process {
+                """
+                input[0] = [
+                    ]
+                input[1] = [
+                    file(params.test_data['sarscov2']['nanopore']['test_fastq_gz'], checkIfExists: true)
+                    ]
+                input[2] = [
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(file(process.out.report_data[0]).readLines()[11..67]).match() },
```

Comment:
Oooh clever! not seen line-range thing! before

Reply:
Thank you, very useful to avoid timestamps haha